### PR TITLE
Update AMP PD Data Explorer to v2 release

### PR DESCRIPTION
**Pull request by wnojopra**
This PR updates the release version from 2019_v1release_1015 to 2020_v2release_1218. Also updates the text indicating number of participants. 

Tested by running locally.

### CODE REVIEW

**Review by panentheos — APPROVED** [review]
Looks good, tested locally that the link works (but I could not test access to the actual dataset)

**Comment by wnojopra** [comment]
Thanks @panentheos . I don't have write access to this repo, so I can't merge this PR. Can someone on your team merge this PR? Thanks!

**Pull request merged**
